packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -265,6 +273,21 @@ export async function activate(context: vscode.ExtensionContext) {
     scratchOrgDecorator.showOrg();
     scratchOrgDecorator.monitorConfigChanges();
   }
+
+  const api: any = {
```

Comment:
We export these using https://code.visualstudio.com/docs/extensionAPI/vscode-api#_extensions

@DatGuyJonathan - I was deciding between the way we discussed and this way. Ultimately I felt this was better since it is more scalable. If we have other packages in the future, we want to allow them to add it in their package and not pollute core.